pages.pt_BR/common/aws-s3.md

```diff
@@ -0,0 +1,28 @@
+# aws s3
+
+> Interface de linha de comando para AWS S3 - provê armazenamento através de uma interface de web services.
```

Comment:
We should replace that `-` from the English page too:
```suggestion
> Interface de linha de comando para AWS S3.
> Provê armazenamento através de uma interface de web services.
```
You can include this tiny edition to the English page in this PR.